widgets/src/components/common/WidgetBox.js

```diff
@@ -1,23 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import withStyles from '@material-ui/core/styles/withStyles';
 import classNames from 'classnames';
-import Paper from '@material-ui/core/Paper';
+import { Paper, Box, Typography, IconButton, Divider } from '@material-ui/core';
+import { ExpandMore as ExpandMoreIcon, ExpandLess as ExpandLessIcon } from '@material-ui/icons';
 
 const styles = {
   widgetBox: {
     background: '#FEFEFE',
-    padding: '20px 25px',
   },
 };
 
-const WidgetBox = ({ classes, passedClassName, children }) => (
-  <Paper variant="outlined" square className={classNames(classes.widgetBox, passedClassName)}>
-    {children}
-  </Paper>
-);
+const WidgetBox = ({
+  title,
+  actions,
```

Comment:
what are `actions`?

Reply:
It's the button/s on the upper right of a widget. But yeah `actions` probably need to be renamed as it's quite vague.